In [46]:
import pandas as pd
import numpy as np
path = "data\\starbucks_drinkMenu_expanded.csv"

data = pd.read_csv(path)
data.head()


Beverage_category       Beverage      Beverage_prep  Calories  \
0                   Coffee  Brewed Coffee              Short         3   
1                   Coffee  Brewed Coffee               Tall         4   
2                   Coffee  Brewed Coffee             Grande         5   
3                   Coffee  Brewed Coffee              Venti         5   
4  Classic Espresso Drinks    Caffè Latte  Short Nonfat Milk        70   

   Total Fat (g)  Trans Fat (g)   Saturated Fat (g)   Sodium (mg)  \
0            0.1             0.0                0.0             0   
1            0.1             0.0                0.0             0   
2            0.1             0.0                0.0             0   
3            0.1             0.0                0.0             0   
4            0.1             0.1                0.0             5   

    Total Carbohydrates (g)   Cholesterol (mg)   Dietary Fibre (g)  \
0                          5                 0                   0   
1                         10                 0                   0   
2                         10                 0                   0   
3                         10                 0                   0   
4                         75                10                   0   

    Sugars (g)   Protein (g)  Vitamin A (% DV)  Vitamin C (% DV)  \
0            0            0.3                0%               0%   
1            0            0.5                0%               0%   
2            0            1.0                0%               0%   
3            0            1.0                0%               0%   
4            9            6.0               10%               0%   

   Calcium (% DV)  Iron (% DV)  Caffeine (mg)  
0               0%           0%           175  
1               0%           0%           260  
2               0%           0%           330  
3               2%           0%           410  
4              20%           0%            75

In [96]:
data

Beverage_category                                      Beverage  \
0                        Coffee                                 Brewed Coffee   
1                        Coffee                                 Brewed Coffee   
2                        Coffee                                 Brewed Coffee   
3                        Coffee                                 Brewed Coffee   
4       Classic Espresso Drinks                                   Caffè Latte   
..                          ...                                           ...   
237  Frappuccino® Blended Crème  Strawberries & Crème (Without Whipped Cream)   
238  Frappuccino® Blended Crème          Vanilla Bean (Without Whipped Cream)   
239  Frappuccino® Blended Crème          Vanilla Bean (Without Whipped Cream)   
240  Frappuccino® Blended Crème          Vanilla Bean (Without Whipped Cream)   
241  Frappuccino® Blended Crème          Vanilla Bean (Without Whipped Cream)   

          Beverage_prep  Calories  Total Fat (g)  Trans Fat (g)   \
0                 Short         3            0.1             0.0   
1                  Tall         4            0.1             0.0   
2                Grande         5            0.1             0.0   
3                 Venti         5            0.1             0.0   
4     Short Nonfat Milk        70            0.1             0.1   
..                  ...       ...            ...             ...   
237             Soymilk       320            3 2             0.4   
238    Tall Nonfat Milk       170            0.1             0.1   
239          Whole Milk       200            3.5             2.0   
240             Soymilk       180            1.5             0.2   
241  Grande Nonfat Milk       240            0.1             0.1   

     Saturated Fat (g)   Sodium (mg)   Total Carbohydrates (g)   \
0                  0.0             0                          5   
1                  0.0             0                         10   
2                  0.0             0                         10   
3                  0.0             0                         10   
4                  0.0             5                         75   
..                 ...           ...                        ...   
237                0.0             0                        250   
238                0.0             0                        160   
239                0.1            10                        160   
240                0.0             0                        160   
241                0.0             5                        230   

     Cholesterol (mg)   Dietary Fibre (g)   Sugars (g)   Protein (g)   \
0                   0                   0            0            0.3   
1                   0                   0            0            0.5   
2                   0                   0            0            1.0   
3                   0                   0            0            1.0   
4                  10                   0            9            6.0   
..                ...                 ...          ...            ...   
237                67                   1           64            5.0   
238                39                   0           38            4.0   
239                39                   0           38            3.0   
240                37                   1           35            3.0   
241                56                   0           55            5.0   

    Vitamin A (% DV)  Vitamin C (% DV)  Calcium (% DV)  Iron (% DV)   \
0                  0%               0%               0%           0%   
1                  0%               0%               0%           0%   
2                  0%               0%               0%           0%   
3                  0%               0%               2%           0%   
4                 10%               0%              20%           0%   
..                ...              ...              ...          ...   
237                6%               8%

In [48]:
partna=data.loc[150:160]


In [76]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 242 entries, 0 to 241
Data columns (total 18 columns):
 #   Column                     Non-Null Count  Dtype  
---  ------                     --------------  -----  
 0   Beverage_category          242 non-null    object 
 1   Beverage                   242 non-null    object 
 2   Beverage_prep              242 non-null    object 
 3   Calories                   242 non-null    int64  
 4    Total Fat (g)             242 non-null    object 
 5   Trans Fat (g)              242 non-null    float64
 6   Saturated Fat (g)          242 non-null    float64
 7    Sodium (mg)               242 non-null    int64  
 8    Total Carbohydrates (g)   242 non-null    int64  
 9   Cholesterol (mg)           242 non-null    int64  
 10   Dietary Fibre (g)         242 non-null    int64  
 11   Sugars (g)                242 non-null    int64  
 12   Protein (g)               242 non-null    float64
 13  Vitamin A (% DV)           242 non-null    object 

In [77]:
partna

Beverage_category                                         Beverage  \
150       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
151       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
152       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
153       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
154  Shaken Iced Beverages          Iced Brewed Coffee (With Classic Syrup)   
155  Shaken Iced Beverages          Iced Brewed Coffee (With Classic Syrup)   
156  Shaken Iced Beverages          Iced Brewed Coffee (With Classic Syrup)   
157  Shaken Iced Beverages   Iced Brewed Coffee (With Milk & Classic Syrup)   
158  Shaken Iced Beverages   Iced Brewed Coffee (With Milk & Classic Syrup)   
159  Shaken Iced Beverages   Iced Brewed Coffee (With Milk & Classic Syrup)   
160  Shaken Iced Beverages   Iced Brewed Coffee (With Milk & Classic Syrup)   

          Beverage_prep  Calories  Total Fat (g)  Trans Fat (g)   \
150             Soymilk       170            3.5             0.4   
151   Venti Nonfat Milk       190            0.2             0.1   
152             2% Milk       230              5             2.5   
153             Soymilk       210              4             0.5   
154                Tall        60              0             0.0   
155              Grande        90            0.1             0.0   
156               Venti       130            0.1             0.0   
157    Tall Nonfat Milk        80            0.1             0.0   
158             2% Milk        90              1             0.5   
159             Soymilk        80              1             0.1   
160  Grande Nonfat Milk       110            0.1             0.0   

     Saturated Fat (g)   Sodium (mg)   Total Carbohydrates (g)   \
150                0.0             0                         80   
151                0.0             5                        110   
152                0.2            20                        125   
153                0.0             0                        100   
154                0.0             0                          4   
155                0.0             0                          5   
156                0.0             0                          5   
157                0.0             0                         25   
158                0.0             5                         25   
159                0.0             0                         20   
160                0.0             0                         30   

     Cholesterol (mg)   Dietary Fibre (g)   Sugars (g)   Protein (g)   \
150                27                   1           25            6.0   
151                39                   0           39            9.0   
152                38                   0           38            9.0   
153                34                   1           32            7.0   
154                15                   0           15            0.2   
155                21                   0           21            0.3   
156                31                   0           31            0.4   
157                18                   0           18            2.0   
158                18                   0           18            2.0   
159                17                   0           17            2.0   
160                24                   0           24            2.0   

    Vitamin A (% DV)  Vitamin C (% DV)  Calcium (% DV)  Iron (% DV)   \
150                8%               0%              25%          10%   
151               15%               0%              30%           0%   
152               15%               0%              30%           0%   
153               10%               0%              30%          15%   
154                0%               0%               0%           0%   
155                0%               0%               0%           0%   
156                0%               0%               0%  

In [89]:
partna=partna.dropna()
partna

Beverage_category                                         Beverage  \
150       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
151       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
152       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
153       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
154  Shaken Iced Beverages          Iced Brewed Coffee (With Classic Syrup)   
155  Shaken Iced Beverages          Iced Brewed Coffee (With Classic Syrup)   
156  Shaken Iced Beverages          Iced Brewed Coffee (With Classic Syrup)   
157  Shaken Iced Beverages   Iced Brewed Coffee (With Milk & Classic Syrup)   
159  Shaken Iced Beverages   Iced Brewed Coffee (With Milk & Classic Syrup)   
160  Shaken Iced Beverages   Iced Brewed Coffee (With Milk & Classic Syrup)   

          Beverage_prep  Calories  Total Fat (g)  Trans Fat (g)   \
150             Soymilk       170            3.5             0.4   
151   Venti Nonfat Milk       190            0.2             0.1   
152             2% Milk       230              5             2.5   
153             Soymilk       210              4             0.5   
154                Tall        60              0             0.0   
155              Grande        90            0.1             0.0   
156               Venti       130            0.1             0.0   
157    Tall Nonfat Milk        80            0.1             0.0   
159             Soymilk        80              1             0.1   
160  Grande Nonfat Milk       110            0.1             0.0   

     Saturated Fat (g)   Sodium (mg)   Total Carbohydrates (g)   \
150                0.0             0                         80   
151                0.0             5                        110   
152                0.2            20                        125   
153                0.0             0                        100   
154                0.0             0                          4   
155                0.0             0                          5   
156                0.0             0                          5   
157                0.0             0                         25   
159                0.0             0                         20   
160                0.0             0                         30   

     Cholesterol (mg)   Dietary Fibre (g)   Sugars (g)   Protein (g)   \
150                27                   1           25            6.0   
151                39                   0           39            9.0   
152                38                   0           38            9.0   
153                34                   1           32            7.0   
154                15                   0           15            0.2   
155                21                   0           21            0.3   
156                31                   0           31            0.4   
157                18                   0           18            2.0   
159                17                   0           17            2.0   
160                24                   0           24            2.0   

    Vitamin A (% DV)  Vitamin C (% DV)  Calcium (% DV)  Iron (% DV)   \
150                8%               0%              25%          10%   
151               15%               0%              30%           0%   
152               15%               0%              30%           0%   
153               10%               0%              30%          15%   
154                0%               0%               0%           0%   
155                0%               0%               0%           0%   
156                0%               0%               0%           0%   
157                2%               0%               6%           0%   
159                2%               0%               6%           0%   
160                4%               0%               8%           2%   

    Caffeine (mg)  
150             0  
151            

In [79]:
varies = data.loc[100:110]

In [80]:
varies


Beverage_category                                     Beverage  \
100  Signature Espresso Drinks  Caramel Apple Spice (Without Whipped Cream)   
101  Signature Espresso Drinks  Caramel Apple Spice (Without Whipped Cream)   
102           Tazo® Tea Drinks                                    Tazo® Tea   
103           Tazo® Tea Drinks                                    Tazo® Tea   
104           Tazo® Tea Drinks                                    Tazo® Tea   
105           Tazo® Tea Drinks                                    Tazo® Tea   
106           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
107           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
108           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
109           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
110           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   

         Beverage_prep  Calories  Total Fat (g)  Trans Fat (g)   \
100             Grande       280              0             0.0   
101              Venti       360              0             0.0   
102              Short         0              0             0.0   
103               Tall         0              0             0.0   
104             Grande         0              0             0.0   
105              Venti         0              0             0.0   
106  Short Nonfat Milk       100            0.1             0.1   
107            2% Milk       120              2             1.0   
108            Soymilk       110            1.5             0.2   
109   Tall Nonfat Milk       160            0.2             0.1   
110            2% Milk       190            3.5             1.5   

     Saturated Fat (g)   Sodium (mg)   Total Carbohydrates (g)   \
100                0.0             0                         20   
101                0.0             0                         25   
102                0.0             0                          0   
103                0.0             0                          0   
104                0.0             0                          0   
105                0.0             0                          0   
106                0.0             0                         50   
107                0.1            10                         55   
108                0.0             0                         45   
109                0.0             5                         80   
110                0.1            15                         90   

     Cholesterol (mg)   Dietary Fibre (g)   Sugars (g)   Protein (g)   \
100                70                   0           65            0.0   
101                89                   0           83            0.0   
102                 0                   0            0            0.0   
103                 0                   0            0            0.0   
104                 0                   0            0            0.0   
105                 0                   0            0            0.0   
106                22                   0           21            4.0   
107                22                   0           21            4.0   
108                20                   0           18            3.0   
109                34                   0           32            6.0   
110                34                   0           32            6.0   

    Vitamin A (% DV)  Vitamin C (% DV)  Calcium (% DV)  Iron (% DV)   \
100                0%               0%               0%           0%   
101                0%               0%               0%           0%   
102                0%               0%               0%           0%   
103                0%               0%               0%           0%   
104                0%               0%               0%           0%   
105                0%               0%               0%           0%   
106                6%               0%              10%           0%   
1

In [87]:
varies=varies.dropna()

In [88]:
varies


Beverage_category                                     Beverage  \
100  Signature Espresso Drinks  Caramel Apple Spice (Without Whipped Cream)   
101  Signature Espresso Drinks  Caramel Apple Spice (Without Whipped Cream)   
106           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
107           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
108           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
109           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
110           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   

         Beverage_prep  Calories  Total Fat (g)  Trans Fat (g)   \
100             Grande       280              0             0.0   
101              Venti       360              0             0.0   
106  Short Nonfat Milk       100            0.1             0.1   
107            2% Milk       120              2             1.0   
108            Soymilk       110            1.5             0.2   
109   Tall Nonfat Milk       160            0.2             0.1   
110            2% Milk       190            3.5             1.5   

     Saturated Fat (g)   Sodium (mg)   Total Carbohydrates (g)   \
100                0.0             0                         20   
101                0.0             0                         25   
106                0.0             0                         50   
107                0.1            10                         55   
108                0.0             0                         45   
109                0.0             5                         80   
110                0.1            15                         90   

     Cholesterol (mg)   Dietary Fibre (g)   Sugars (g)   Protein (g)   \
100                70                   0           65            0.0   
101                89                   0           83            0.0   
106                22                   0           21            4.0   
107                22                   0           21            4.0   
108                20                   0           18            3.0   
109                34                   0           32            6.0   
110                34                   0           32            6.0   

    Vitamin A (% DV)  Vitamin C (% DV)  Calcium (% DV)  Iron (% DV)   \
100                0%               0%               0%           0%   
101                0%               0%               0%           0%   
106                6%               0%              10%           0%   
107                6%               0%              10%           0%   
108                4%               0%              10%           6%   
109               10%               0%              20%           2%   
110               10%               0%              20%           2%   

    Caffeine (mg)  
100             0  
101             0  
106            50  
107            50  
108            50  
109            70  
110            70

In [90]:
data=data.dropna()
data

Beverage_category                                      Beverage  \
0                        Coffee                                 Brewed Coffee   
1                        Coffee                                 Brewed Coffee   
2                        Coffee                                 Brewed Coffee   
3                        Coffee                                 Brewed Coffee   
4       Classic Espresso Drinks                                   Caffè Latte   
..                          ...                                           ...   
237  Frappuccino® Blended Crème  Strawberries & Crème (Without Whipped Cream)   
238  Frappuccino® Blended Crème          Vanilla Bean (Without Whipped Cream)   
239  Frappuccino® Blended Crème          Vanilla Bean (Without Whipped Cream)   
240  Frappuccino® Blended Crème          Vanilla Bean (Without Whipped Cream)   
241  Frappuccino® Blended Crème          Vanilla Bean (Without Whipped Cream)   

          Beverage_prep  Calories  Total Fat (g)  Trans Fat (g)   \
0                 Short         3            0.1             0.0   
1                  Tall         4            0.1             0.0   
2                Grande         5            0.1             0.0   
3                 Venti         5            0.1             0.0   
4     Short Nonfat Milk        70            0.1             0.1   
..                  ...       ...            ...             ...   
237             Soymilk       320            3 2             0.4   
238    Tall Nonfat Milk       170            0.1             0.1   
239          Whole Milk       200            3.5             2.0   
240             Soymilk       180            1.5             0.2   
241  Grande Nonfat Milk       240            0.1             0.1   

     Saturated Fat (g)   Sodium (mg)   Total Carbohydrates (g)   \
0                  0.0             0                          5   
1                  0.0             0                         10   
2                  0.0             0                         10   
3                  0.0             0                         10   
4                  0.0             5                         75   
..                 ...           ...                        ...   
237                0.0             0                        250   
238                0.0             0                        160   
239                0.1            10                        160   
240                0.0             0                        160   
241                0.0             5                        230   

     Cholesterol (mg)   Dietary Fibre (g)   Sugars (g)   Protein (g)   \
0                   0                   0            0            0.3   
1                   0                   0            0            0.5   
2                   0                   0            0            1.0   
3                   0                   0            0            1.0   
4                  10                   0            9            6.0   
..                ...                 ...          ...            ...   
237                67                   1           64            5.0   
238                39                   0           38            4.0   
239                39                   0           38            3.0   
240                37                   1           35            3.0   
241                56                   0           55            5.0   

    Vitamin A (% DV)  Vitamin C (% DV)  Calcium (% DV)  Iron (% DV)   \
0                  0%               0%               0%           0%   
1                  0%               0%               0%           0%   
2                  0%               0%               0%           0%   
3                  0%               0%               2%           0%   
4                 10%               0%              20%           0%   
..                ...              ...              ...          ...   
237                6%               8%

In [97]:
data.loc[150:160]

Beverage_category                                         Beverage  \
150       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
151       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
152       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
153       Tazo® Tea Drinks  Tazo® Full-Leaf Red Tea Latte (Vanilla Rooibos)   
154  Shaken Iced Beverages          Iced Brewed Coffee (With Classic Syrup)   
155  Shaken Iced Beverages          Iced Brewed Coffee (With Classic Syrup)   
156  Shaken Iced Beverages          Iced Brewed Coffee (With Classic Syrup)   
157  Shaken Iced Beverages   Iced Brewed Coffee (With Milk & Classic Syrup)   
159  Shaken Iced Beverages   Iced Brewed Coffee (With Milk & Classic Syrup)   
160  Shaken Iced Beverages   Iced Brewed Coffee (With Milk & Classic Syrup)   

          Beverage_prep  Calories  Total Fat (g)  Trans Fat (g)   \
150             Soymilk       170            3.5             0.4   
151   Venti Nonfat Milk       190            0.2             0.1   
152             2% Milk       230              5             2.5   
153             Soymilk       210              4             0.5   
154                Tall        60              0             0.0   
155              Grande        90            0.1             0.0   
156               Venti       130            0.1             0.0   
157    Tall Nonfat Milk        80            0.1             0.0   
159             Soymilk        80              1             0.1   
160  Grande Nonfat Milk       110            0.1             0.0   

     Saturated Fat (g)   Sodium (mg)   Total Carbohydrates (g)   \
150                0.0             0                         80   
151                0.0             5                        110   
152                0.2            20                        125   
153                0.0             0                        100   
154                0.0             0                          4   
155                0.0             0                          5   
156                0.0             0                          5   
157                0.0             0                         25   
159                0.0             0                         20   
160                0.0             0                         30   

     Cholesterol (mg)   Dietary Fibre (g)   Sugars (g)   Protein (g)   \
150                27                   1           25            6.0   
151                39                   0           39            9.0   
152                38                   0           38            9.0   
153                34                   1           32            7.0   
154                15                   0           15            0.2   
155                21                   0           21            0.3   
156                31                   0           31            0.4   
157                18                   0           18            2.0   
159                17                   0           17            2.0   
160                24                   0           24            2.0   

    Vitamin A (% DV)  Vitamin C (% DV)  Calcium (% DV)  Iron (% DV)   \
150                8%               0%              25%          10%   
151               15%               0%              30%           0%   
152               15%               0%              30%           0%   
153               10%               0%              30%          15%   
154                0%               0%               0%           0%   
155                0%               0%               0%           0%   
156                0%               0%               0%           0%   
157                2%               0%               6%           0%   
159                2%               0%               6%           0%   
160                4%               0%               8%           2%   

    Caffeine (mg)  
150             0  
151            

In [98]:
data.loc[100:110]

Beverage_category                                     Beverage  \
100  Signature Espresso Drinks  Caramel Apple Spice (Without Whipped Cream)   
101  Signature Espresso Drinks  Caramel Apple Spice (Without Whipped Cream)   
106           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
107           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
108           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
109           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   
110           Tazo® Tea Drinks                         Tazo® Chai Tea Latte   

         Beverage_prep  Calories  Total Fat (g)  Trans Fat (g)   \
100             Grande       280              0             0.0   
101              Venti       360              0             0.0   
106  Short Nonfat Milk       100            0.1             0.1   
107            2% Milk       120              2             1.0   
108            Soymilk       110            1.5             0.2   
109   Tall Nonfat Milk       160            0.2             0.1   
110            2% Milk       190            3.5             1.5   

     Saturated Fat (g)   Sodium (mg)   Total Carbohydrates (g)   \
100                0.0             0                         20   
101                0.0             0                         25   
106                0.0             0                         50   
107                0.1            10                         55   
108                0.0             0                         45   
109                0.0             5                         80   
110                0.1            15                         90   

     Cholesterol (mg)   Dietary Fibre (g)   Sugars (g)   Protein (g)   \
100                70                   0           65            0.0   
101                89                   0           83            0.0   
106                22                   0           21            4.0   
107                22                   0           21            4.0   
108                20                   0           18            3.0   
109                34                   0           32            6.0   
110                34                   0           32            6.0   

    Vitamin A (% DV)  Vitamin C (% DV)  Calcium (% DV)  Iron (% DV)   \
100                0%               0%               0%           0%   
101                0%               0%               0%           0%   
106                6%               0%              10%           0%   
107                6%               0%              10%           0%   
108                4%               0%              10%           6%   
109               10%               0%              20%           2%   
110               10%               0%              20%           2%   

    Caffeine (mg)  
100             0  
101             0  
106            50  
107            50  
108            50  
109            70  
110            70

In [100]:
data.loc[150:160].isnull()

Beverage_category  Beverage  Beverage_prep  Calories   Total Fat (g)  \
150              False     False          False     False           False   
151              False     False          False     False           False   
152              False     False          False     False           False   
153              False     False          False     False           False   
154              False     False          False     False           False   
155              False     False          False     False           False   
156              False     False          False     False           False   
157              False     False          False     False           False   
159              False     False          False     False           False   
160              False     False          False     False           False   

     Trans Fat (g)   Saturated Fat (g)   Sodium (mg)  \
150           False              False         False   
151           False              False         False   
152           False              False         False   
153           False              False         False   
154           False              False         False   
155           False              False         False   
156           False              False         False   
157           False              False         False   
159           False              False         False   
160           False              False         False   

      Total Carbohydrates (g)   Cholesterol (mg)   Dietary Fibre (g)  \
150                      False             False               False   
151                      False             False               False   
152                      False             False               False   
153                      False             False               False   
154                      False             False               False   
155                      False             False               False   
156                      False             False               False   
157                      False             False               False   
159                      False             False               False   
160                      False             False               False   

      Sugars (g)   Protein (g)   Vitamin A (% DV)   Vitamin C (% DV)  \
150        False          False              False             False   
151        False          False              False             False   
152        False          False              False             False   
153        False          False              False             False   
154        False          False              False             False   
155        False          False              False             False   
156        False          False              False             False   
157        False          False              False             False   
159        False          False              False             False   
160        False          False              False             False   

      Calcium (% DV)   Iron (% DV)   Caffeine (mg)  
150             False         False          False  
151             False         False          False  
152             False         False          False  
153             False         False          False  
154             False         False          False  
155             False         False          False  
156             False         False          False  
157             False         False          False  
159             False         False          False  
160             False         False          False

In [101]:
data.loc[100:110].isnull()

Beverage_category  Beverage  Beverage_prep  Calories   Total Fat (g)  \
100              False     False          False     False           False   
101              False     False          False     False           False   
106              False     False          False     False           False   
107              False     False          False     False           False   
108              False     False          False     False           False   
109              False     False          False     False           False   
110              False     False          False     False           False   

     Trans Fat (g)   Saturated Fat (g)   Sodium (mg)  \
100           False              False         False   
101           False              False         False   
106           False              False         False   
107           False              False         False   
108           False              False         False   
109           False              False         False   
110           False              False         False   

      Total Carbohydrates (g)   Cholesterol (mg)   Dietary Fibre (g)  \
100                      False             False               False   
101                      False             False               False   
106                      False             False               False   
107                      False             False               False   
108                      False             False               False   
109                      False             False               False   
110                      False             False               False   

      Sugars (g)   Protein (g)   Vitamin A (% DV)   Vitamin C (% DV)  \
100        False          False              False             False   
101        False          False              False             False   
106        False          False              False             False   
107        False          False              False             False   
108        False          False              False             False   
109        False          False              False             False   
110        False          False              False             False   

      Calcium (% DV)   Iron (% DV)   Caffeine (mg)  
100             False         False          False  
101             False         False          False  
106             False         False          False  
107             False         False          False  
108             False         False          False  
109             False         False          False  
110             False         False          False

The data has been manipulated and corrected, in that non-numerical values such as "NaN" or "Varies" in columns that are supposed to be filled by solely numerical values have been dropped from the data table. We chose to manipulate Starbucks drink menu data because this is a sector that many people can relate to and would benefit from receiving clean and readable data. The data happens to be quite recent and is very interesting to report on, in that there was a single entry that was never filled by official employees relating to the drink menu database. 
"Beverage_category" qualitatively describes the type of beverage made, "Beverage" is the name of the beverage coined by Starbucks, and "Beverage_prep" details about the size of the beverage, and the rest of the columns approaching towards the right describe the quantitative, numerical data about the nutritional facts of the beverages respectively. Our aim was to manipulate and clean up the data, dropping any values that were non-numerical in the numerical data columns, which was successfully accomplished. 
By importing pandas, we were able to read the dataframe/csv file (the data table) from Excel and manipulate the data. By using dropna() and .loc() functionalities, irrespectively, we were able locate missing data or data that was non-numerical in our numerically-desired columns, and effectively drop these values. These values could have also been filled with the mean of the respective columns and datasets or with a certain keyword using var=data.fillna(data.mean()) or var=data.fillna("placeholder") if assigning a local variable derived from "data", respectively. 
In the end, it was found that there were 237 non-null rows, whereas initially there were 242 rows in total. This indicates that we dropped 5 null or non-numerically desired rows. By manually parsing through data in Excel, and confirming this using .loc, the non-numerical/NaN values were located in rows 102-105 and 158. These were subsequently dropped.
This assignment helped us understand how to easily manipulate data from data sets in an effective manner using Python and its libraries.